pkg/cmd/server/origin/openshift_apiserver.go

```diff
@@ -189,6 +191,12 @@ type legacyStorageMutator interface {
 	mutate(map[schema.GroupVersion]map[string]rest.Storage)
 }
 
+type legacyStorageMutatorFunc func(map[schema.GroupVersion]map[string]rest.Storage)
```

Comment:
maybe worth a little godoc explaining why is this needed and if/when this will go away?

Reply:
also is this apps only or do we need this in other api servers?

Reply:
> maybe worth a little godoc explaining why is this needed and if/when this will go away?

I'm not sure of the final factorization. This pattern began a couple apiservers back.  `users` I think.

Reply:
> also is this apps only or do we need this in other api servers?

The interface is doced